docs/sources/installation/mac.md

```diff
@@ -10,37 +10,61 @@ weight = 500
 
 # Install on macOS
 
-This page provides instructions to help you install Grafana on macOS. 
+This page explains how to install Grafana and get the service running on your macOS. It has the following topics:  
+   - [Install with Homebrew](#install-with-homebrew)
+   - [Install standalone macOS binaries](#install-standalone-macos-binaries)
+   - [Next steps](#next-steps)
+   - [Upgrade](#upgrade)
 
-**Note on upgrading:** While the process for upgrading Grafana is very similar to installing Grafana, there are some key backup steps you should perform. Read [Upgrading Grafana]({{< relref "upgrading.md" >}}) for tips and guidance on updating an existing installation.
+**Note on upgrading:** While the process for upgrading Grafana is very similar to installing Grafana, there are some key backup steps you should perform. Before you perform an upgrade, read [Upgrading Grafana]({{< relref "upgrading.md" >}}) for tips and guidance on updating an existing installation.
 
-## 1. Download and install
+## Install with Homebrew
 
-Before you begin, you must have [Homebrew](http://brew.sh/) installed.
+Use [Homebrew](http://brew.sh/) to install the most recent released version of Grafana using Homebrew package.
+
+1. On the Homberew homepage, search for Grafana. The last stable and released version is listed.  
+1. Open a terminal and enter:
+    ```
+   brew update
+   brew install grafana
+   ```
+
+   The brew page downloads and untars the files into `/usr/local/Cellar/grafana/version`.
+
+1. Start Grafana using the command:
+   ```bash
+   brew services start grafana
+   ```
+
+## Install standalone macOS binaries
+
+To install a nightly build, or to install the latest version of Grafana  without Homebrew, go to the [Grafana download page](https://grafana.com/grafana/download/7.3.0-381ff45epre?platform=mac).
+
+1. Select the Grafana version you want to install. By default, the most recent released version is selected.
+
+   > **Note:** The downloads page lists only finished releases. If you want to install a beta version, click [Nightly ] **Nightly Builds** and then select a version.
 
-1. On the [Grafana download page](https://grafana.com/grafana/download?platform=mac), select the Grafana version you want to install. 
-   * The most recent Grafana version is selected by default.
-   * The **Version** field displays only finished releases. If you want to install a beta version, click **Nightly Builds** and then select a version.
 2. Select an **Edition**.
```

Comment:
Please change this numbering to lazy numbering.

Reply:
Fixed.